recipe/meta.yaml

```diff
@@ -36,17 +36,17 @@ requirements:
     - libwebp-base 1.2.4 # [linux or osx]
     - xz 5.2.*
     - zlib 1.2.13  # should be actually 1.2.11, but pinning hell
-    - zstd 1.5.2
-    - lerc 3.0
-    - libdeflate 1.17
+    - zstd {{ zstd }}
+    - lerc 4
+    - libdeflate 1.22
   run:
     - jpeg >=9e,<10a
     - libwebp-base >=1.2.4,<2.0a0 # [linux or osx]
-    - xz >=5.2.4,<6.0a0
+    - xz
     - zlib >=1.2.13,<1.3.0a0
     - zstd >=1.5.2,<1.6.0a0
-    - lerc >=3.0,<4.0a0
-    - libdeflate >=1.17,<1.18.0a0
```

Comment:
See the difference:
- before https://staging.continuum.io/package/prefect/fs/libtiff-feedstock/pr20/545dac2/linux-64/libtiff-4.5.1-h80c7b02_1.conda
- after https://staging.continuum.io/package/prefect/fs/libtiff-feedstock/pr20/dcd043c/linux-64/libtiff-4.5.1-hffd6297_1.conda